app/src/main/java/com/genesis/ai/app/ipc/AuraDriveServiceImpl.kt

```
@@ -63,20 +84,23 @@ class AuraDriveServiceImpl : Service() {
+            val enabledModules = moduleStates.filterValues { it }.keys
+
             val jsonObject = JsonObject().apply {
                 addProperty("timestamp", System.currentTimeMillis())
                 addProperty("oracleDriveAppVersion", appVersionName)
-                addProperty("isRooted", checkRootStatus()) // Internal helper
+                addProperty("isRooted", checkRootStatus())
                 addProperty("containerOSVersion", Build.VERSION.RELEASE)
                 addProperty("containerSDKVersion", Build.VERSION.SDK_INT)
                 val backendStatus = try {
                     if (GenesisRepositoryNew.api != null) "Available" else "Unavailable"
                 } catch (e: Exception) { "Error" }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)